proposals/specification/contexts/social-agent-profile.jsonld

```diff
@@ -0,0 +1,26 @@
+{
+  "@context": {
+    "@version": 1.1,
+    "@protected": true,
+    "authorization_agent": {
+      "@id": "http://www.w3.org/ns/solid/interop#hasAuthorizationAgent",
+      "@type": "@id"
+    },
+    "access_inbox": {
+      "@id": "http://www.w3.org/ns/solid/interop#hasAccessInbox",
+      "@type": "@id"
+    },
+    "registry_set": {
+      "@id": "http://www.w3.org/ns/solid/interop#hasRegistrySet",
+      "@type": "@id"
+    },
+    "oidc_issuer": {
```

Comment:
Why `oidc_issuer` and not `oidcIssuer`? That seems atypical. Same applies to the other properties.

Reply:
no strong preference one way or the other. most json-ld that I've seen uses this style, including solid-oidc context, so I stuck with that. not against camelcasing.